src/compiler/tree.cc

```diff
@@ -274,6 +316,10 @@ void TreeGrower::grow(Program* program) {
     logger = &null_logger;
   }
 
+  // Selectors for which the dynamic call was changed to a static call to
+  // one of the methods in the set.
+  Map<CallSelector, UnorderedSet<Method*>> filtered_selectors;
```

Comment:
I don't like the name of this so much. I think this is easier to understand as a set of methods where we changed a dynamic call to a static call. I think adding an abstraction around this (`StaticallyCalledInstanceMethodSet`) could be nice.

Reply:
changed.